.travis.yml

```diff
@@ -83,7 +83,7 @@ script:
       python examples/seismic/benchmark.py test -P tti -so 4 -a -d 20 20 20 -n 5;
       python examples/seismic/benchmark.py test -P acoustic -a;
       python examples/seismic/acoustic/acoustic_example.py --full;
-      python examples/seismic/acoustic/constant_example.py --full;
+      python examples/seismic/acoustic/acoustic_example.py -c;
```

Comment:
should we keep both ``--full`` and ``-c`` ?

Reply:
Yes, two different things, `full` is to run all operators (born, adjoint, gradient)